src/Uno.UI.FluentTheme.v2/themeresources_v2.xaml

```diff
@@ -22960,6 +22960,10 @@
                 <ColumnDefinition Width="*" />
                 <ColumnDefinition Width="Auto" />
               </Grid.ColumnDefinitions>
+              <!-- Uno workaround: On Android, the ContentPresenters inside this are not calculating correctly -->
+              <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+              </Grid.RowDefinitions>
```

Comment:
Let's scope to android using XAML conditions.

Reply:
Done!